pipelines/build/common/build_base_file.groovy

```diff
@@ -181,7 +181,7 @@ class Builder implements Serializable {
             ENABLE_TESTDYNAMICPARALLEL: enableTestDynamicParallel,
             ENABLE_INSTALLERS: enableInstallers,
             ENABLE_SIGNER: enableSigner,
-            CLEAN_WORKSPACE: cleanWorkspace,
+            CLEAN_WORKSPACE_BEFORE: cleanWorkspaceBefore,
```

Comment:
#134 def cleanWorkspace = cleanWorkspaceBeforeBuild and #136 need to be updated.

Reply:
@zdtsw I have fixed this